test/functional/plugin/lsp_spec.lua

```diff
@@ -624,6 +624,37 @@ describe('LSP', function()
       eq(true, result.detach_called)
     end)
 
+    it('should detach buffer on bufwipe 2', function()
```

Comment:
why do we need 2 tests, can we delete the old `'should detach buffer on bufwipe'` test above this one?

Reply:
removed old test in https://github.com/neovim/neovim/pull/35364